python/beeai_framework/adapters/acp/agents/agent.py

```diff
@@ -15,88 +15,90 @@
 from functools import reduce
 
 try:
-    from acp_sdk.client import Client
-    from acp_sdk.models import Message, MessagePart, RunCompletedEvent, RunFailedEvent
-    from acp_sdk.models.errors import Error
+    import acp_sdk.client as acp_client
+    import acp_sdk.models as acp_models
+
+    from beeai_framework.adapters.acp.agents.events import (
+        ACPAgentUpdateEvent,
+        ACPErrorEvent,
+        beeai_platform_agent_event_types,
+    )
+    from beeai_framework.adapters.acp.agents.types import (
+        ACPAgentRunOutput,
+    )
 except ModuleNotFoundError as e:
     raise ModuleNotFoundError(
-        "Optional module [acp] not found.\nRun 'pip install \"beeai-framework[acp]\"' to install."
+        "Optional module [beeai-platform] not found.\nRun 'pip install \"beeai-framework[beeai-platform]\"' to install."
```

Comment:
I don't see this update in the pyproject.toml in this PR, did you forget to include it?